presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeRemoteFunctions.java

```diff
@@ -49,12 +50,10 @@ protected void preInitQueryRunners()
     {
         // Initialize the remote function thrift server process.
         String propertyName = "REMOTE_FUNCTION_SERVER";
-        remoteFunctionServerBinaryPath = System.getenv(propertyName);
-        if (remoteFunctionServerBinaryPath == null) {
-            remoteFunctionServerBinaryPath = System.getProperty(propertyName);
-        }
+        remoteFunctionServerBinaryPath = getProperty(propertyName).orElse(null);
```

Comment:
I think it's better to throw errors when the system property needs to be set. Maybe use `orElseThrow` here?

```suggestion
        remoteFunctionServerBinaryPath = getProperty(propertyName).orElseThrow(() -> ...);
```

Reply:
UPdated